### ISSUE TITLE
java.lang.NoSuchMethodError: void kotlinx.metadata.internal.ReadersKt.accept$default

### ISSUE THREAD

**Issue by CheeseTastisch** [ProjectMapK/jackson-module-kogera#214]
When i try to serialize the following class: 
```kotlin
class SettingUpdatePacket(val key: String, val value: String) : Packet

interface Packet
```

i get the following error:
```
java.lang.NoSuchMethodError: 'void kotlinx.metadata.internal.ReadersKt.accept$default(kotlinx.metadata.internal.metadata.ProtoBuf$Class, kotlinx.metadata.KmClassVisitor, kotlinx.metadata.internal.metadata.deserialization.NameResolver, java.util.List, int, java.lang.Object)'
        at io.github.projectmapk.jackson.module.kogera.JmClassKt.accept(JmClass.kt:207)
        at io.github.projectmapk.jackson.module.kogera.JmClassKt.access$accept(JmClass.kt:1)
        at io.github.projectmapk.jackson.module.kogera.JmClassImpl.<init>(JmClass.kt:129)
        at io.github.projectmapk.jackson.module.kogera.JmClassKt.JmClass(JmClass.kt:215)
        at io.github.projectmapk.jackson.module.kogera.ReflectionCache.getJmClass(ReflectionCache.kt:64)
        at io.github.projectmapk.jackson.module.kogera.ReflectionCache.getJmClass(ReflectionCache.kt:62)
        at io.github.projectmapk.jackson.module.kogera.annotationIntrospector.KotlinFallbackAnnotationIntrospector.findImplicitPropertyName(KotlinFallbackAnnotationIntrospector.kt:58)
        at com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair.findImplicitPropertyName(AnnotationIntrospectorPair.java:488)
        at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector._addGetterMethod(POJOPropertiesCollector.java:820)
        at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector._addMethods(POJOPropertiesCollector.java:761)
        at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector.collectAll(POJOPropertiesCollector.java:447)
        at com.fasterxml.jackson.databind.introspect.POJOPropertiesCollector.getJsonValueAccessor(POJOPropertiesCollector.java:286)
        at com.fasterxml.jackson.databind.introspect.BasicBeanDescription.findJsonValueAccessor(BasicBeanDescription.java:258)
        at com.fasterxml.jackson.databind.ser.BasicSerializerFactory.findSerializerByAnnotations(BasicSerializerFactory.java:393)
        at com.fasterxml.jackson.databind.ser.BeanSerializerFactory._createSerializer2(BeanSerializerFactory.java:225)
        at com.fasterxml.jackson.databind.ser.BeanSerializerFactory.createSerializer(BeanSerializerFactory.java:174)
        at com.fasterxml.jackson.databind.SerializerProvider._createUntypedSerializer(SerializerProvider.java:1503)
        at com.fasterxml.jackson.databind.SerializerProvider._createAndCacheUntypedSerializer(SerializerProvider.java:1451)
        at com.fasterxml.jackson.databind.SerializerProvider.findValueSerializer(SerializerProvider.java:556)
        at com.fasterxml.jackson.databind.SerializerProvider.findTypedValueSerializer(SerializerProvider.java:834)
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:307)
        at com.fasterxml.jackson.databind.ObjectMapper.valueToTree(ObjectMapper.java:3532)
        [...]
```

How i try to serialize:
```kotlin
mapper.valueToTree(value)
```

**Comment by k163377** [comment]
The following code could not be reproduced.

```kotlin
import org.junit.jupiter.api.Test

class GitHub214SettingUpdatePacket {
    class SettingUpdatePacket(val key: String, val value: String) : Packet

    interface Packet

    @Test
    fun test() {
        val mapper = defaultMapper

        val dto = SettingUpdatePacket("k", "v")

        println(mapper.writeValueAsString(dto))
        println(mapper.valueToTree(dto))
    }
}
```

It probably looks like a `kotlinx-metadata-jvm` version mismatch.
Currently `0.7.0` is used in this project, the latest is `0.9.0`.

**Comment by CheeseTastisch** [comment]
Thanks! After chaing my included kotlinx-metadata-jvm version to 0.7.0 it worked.

**Comment by k163377** [comment]
Update to `kotlinx-metadata-jvm 0.9.0` is now available.
https://github.com/ProjectMapK/jackson-module-kogera/releases/tag/2.16.1-beta11
